clients/spark/core/src/main/scala/io/treeverse/clients/LakeFSInputFormat.scala

```diff
@@ -1,74 +1,133 @@
 package io.treeverse.clients
 
-import io.treeverse.lakefs.catalog.Entry
 import io.treeverse.clients.LakeFSContext._
+import io.treeverse.lakefs.catalog.Entry
 import io.treeverse.lakefs.graveler.committed.RangeData
+import org.apache.commons.lang3.StringUtils
+import org.apache.hadoop.fs.FileSystem
 import org.apache.hadoop.fs.Path
 import org.apache.hadoop.io.Writable
 import org.apache.hadoop.mapred.SplitLocationInfo
 import org.apache.hadoop.mapreduce._
-import scalapb.{GeneratedMessage, GeneratedMessageCompanion}
+import scalapb.GeneratedMessage
+import scalapb.GeneratedMessageCompanion
 
-import java.io.{DataInput, DataOutput, File}
+import java.io.DataInput
+import java.io.DataOutput
+import java.io.File
+import java.net.URI
 import scala.collection.JavaConverters._
+import scala.collection.mutable.ListBuffer
+import com.google.protobuf.ByteString
+import org.slf4j.{Logger, LoggerFactory}
 
-class GravelerSplit(var range: RangeData, var path: Path) extends InputSplit with Writable {
-  def this() = this(null, null)
+object GravelerSplit {
+  val logger: Logger = LoggerFactory.getLogger(getClass.toString)
+}
+class GravelerSplit(var range: RangeData, var path: Path, var rangeID: String, var byteSize: Long)
+    extends InputSplit
+    with Writable {
+  import GravelerSplit._
+  def this() = this(null, null, null, 0L)
```

Comment:
I don't understand this (sorry, bad pun, but the point is valid).  Scala already has a `this` operator, and this makes it really hard to understand to what `this` refers.  Also it does not make sense for most purposes.  Could you document it, or change the name?